src/nu/validator/checker/schematronequiv/Assertions.java

```diff
@@ -24,17 +24,8 @@
 
 import java.io.ByteArrayInputStream;
 import java.io.StringReader;
+import java.util.*;
```

Comment:
```suggestion
import java.util.*;
```
We intentionally have never used any wildcard imports in the sources for the checker (and the htmlparser); it’s not an accident or oversight that we haven’t.

So unless there’s some very compelling reason to use a wildcard import here, let’s please instead stick with explicitly listing everything we need.

Reply:
That's just automatic change by my IDE. I have commited a change.